Hexastore.Web/EventHubs/EventSender.cs

```diff
@@ -50,10 +49,15 @@ public async Task SendMessage(StoreEvent storeEvent)
             }
 
             try {
-                storeEvent.PartitionId = storeEvent.StoreId.GetHashCode() % _storeConfig.EventHubPartitionCount;
+                int ehPartitionId;
+                if (!string.IsNullOrEmpty(storeEvent.PartitionId)) {
+                    ehPartitionId = storeEvent.PartitionId.GetHashCode() % _storeConfig.EventHubPartitionCount;
+                } else {
+                    ehPartitionId = storeEvent.StoreId.GetHashCode() % _storeConfig.EventHubPartitionCount;
+                }
                 var content = JsonConvert.SerializeObject(storeEvent, Formatting.None);
                 var bytes = Encoding.UTF8.GetBytes(content);
-                await _eventHubClient.SendAsync(new EventData(bytes), storeEvent.PartitionId.ToString());
+                await _eventHubClient.SendAsync(new EventData(bytes), ehPartitionId.ToString());
```

Comment:
In the docs, this doesn't send to partition ID (the specific partition you want it to go to, like partition 11), but rather the partition key (like a jump hash) that all messages with the same key guarantee go to the same partition, but you don't know which. It's admittedly a nit, but probably good we don't conflate the two and get confused down the road since EH has different meanings for them

Reply:
Looks like all send API's only take partitionKey not partitionId. So out hashing is not that useful. I will remove it.